app/javascript/styles/_global.scss

```diff
@@ -480,6 +480,100 @@ h4, .h4, h5, .h5, h6, .h6 {
   padding-left: 15px;
 }
 
```

Comment:
let's pull these out into a separate _search.scss file and import it into application.scss

Reply:
also, you can make this SCSS more concise/robust by wrapping everything in  a single
#search-panel {} rather than repeating it for each rule

Reply:
[Nesting](https://sass-lang.com/guide#topic-3), wow!  Done in 9576866 and 0873e98.